### PR TITLE
logan merged with sophie's code

### PR DESCRIPTION
**Pull request by logan-qiu**

### CODE REVIEW

File: pages/profile.js [hunk 1]
```diff
@@ -11,34 +14,64 @@ const Profile = () => {
...
+  // const getData = () => {
+  //   axios.get('/api/getUserInfo', { email: "pillsbury.doughboy@gmail.com" })
+  //     .then((response) => {
+  //       // console.log(response.data);
+  //       setUserinfo(response.data);
+  //     })
+  //     .catch((err) => {
+  //       console.log('fetch err: ', err);
+  //     })
+  // }
```
Comment (matt-heindel) [inline pages/profile.js]: Is this cruft?

File: pages/profile.js [hunk 2]
```diff
@@ -11,34 +14,64 @@ const Profile = () => {
...
+
+  useEffect(() => {
+    axios.get('/api/getUserInfo', { params: {email: "pillsbury.doughboy@gmail.com" }})
```
Comment (robin-son1) [inline pages/profile.js:21]: do we have email set up in context since it's currently hardcoded to doughboy?

File: pages/new-post.js [hunk 3]
```diff
@@ -71,7 +58,17 @@ const NewPost = () => {
           <Textarea onChange={handleTxt} placeholder="Please feel free to share your ideas!" />
         </FormControl>
         <br></br>
-        <Confirm colorScheme="red" done={(!top || !cat ? false: true)} handleSubmit={handleSubmit} />
```
Comment (robin-son1) [inline pages/new-post.js:74]: Why are we removing the submit button here?

File: pages/api/getTrendingPosts.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { getTrendingPosts } from '../../lib/models';
 
 export default function handler(req, res) {
   return new Promise(resolve => {
-    getTrendingPosts(req.body.category)
+    getTrendingPosts(req.query.category)
```
Comment (matt-heindel) [inline pages/api/getTrendingPosts.js:5]: thanks for fixing this

**Review by robin-son1 — CHANGES_REQUESTED** [review]
just one comment about the useeffect being hardcoded to a user

File: pages/profile.js (hunk 2 above)
Comment (logan-qiu) [reply pages/profile.js:21]: Hard-coded :/

File: components/new-post/Confirm.jsx [hunk 5]
```diff
@@ -31,9 +31,21 @@ const Confirm = (props) => {
...
+  //     brand: {
+  //       100: '#FF6E40',
+  //       200: '#EBEBEB',
+  //       300: ' #484848',
+  //       400: '#1E3D59',
+  //       500: '#fff',
+  //     }
+  //   }
+  // })
+
```
Comment (matt-heindel) [inline components/new-post/Confirm.jsx:45]: Is this cruft?

File: pages/profile.js (hunk 2 above)
Comment (matt-heindel) [reply pages/profile.js:21]: thanks for getting rid of this!

**Review by matt-heindel — APPROVED** [review]

**Pull request merged**